third_party/pcsc-lite/naclport/server/build/Makefile

```diff
@@ -61,6 +61,7 @@ COMMON_CPPFLAGS := \
 	-I$(SOURCES_PATH) \
 	-I$(PCSC_LITE_ORIGINAL_HEADERS_DIR_PATH) \
 	-I$(PCSC_LITE_SOURCES_PATH) \
+	-I$(ROOT_PATH) \
 	-I$(ROOT_PATH)/common/cpp/src \
```

Comment:
Are both of these lines required?

Reply:
For now, yes. Eventually my plan is to only leave `$(ROOT_PATH)` and get rid of most of other paths, but will tackle this step-by-step.